kbhff/test/fixtures.py

```diff
@@ -7,11 +7,10 @@
 
 from pyvirtualdisplay import Display
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="function")
 def gmail():
-    gmail = get_gmail_connection()
+    gmail = GmailConnection()
     yield gmail
```

Comment:
Do you need to explicitly `del gmail` at the end of this, or does python free it automatically? I would guess the latter, in which case you don't need `del gmail` in the `test_canLogIntoGmail` test, I think.